app/config/parameters.yml.dist

```diff
@@ -14,4 +14,5 @@ parameters:
     mailer_password: ~
 
     secret: EDITME
+    defuse_secret: def000003f125cbd07bfce7748bf501fa3f5ebb6b8a4d785ddb7b328948ea4e88973d5c1cdf79a5a2784b723ad2fb9d802111d73ec98be7ef55e47966a89e9b1fcfa35b0
```

Comment:
There are three aproaches here:
 - do not provide default defuse secret (so app will crash when trying to use `EDITME` defuse secret key)
 - provide default defuse secret key (so it would change nothing from the current situation)
 - do not encrypt gateway configs by default and provide a cookbook (so less people will even know about it)